urwid/event_loop/asyncio_loop.py

```diff
@@ -97,38 +137,42 @@ def remove_watch_file(self, handle: int) -> bool:
         """
         return self._loop.remove_reader(handle)
 
-    def enter_idle(self, callback: Callable[[], typing.Any]) -> list[asyncio.TimerHandle]:
+    def enter_idle(self, callback: Callable[[], typing.Any]) -> int:
         """
         Add a callback for entering idle.
 
-        Returns a handle that may be passed to remove_idle()
+        Returns a handle that may be passed to remove_enter_idle()
         """
         # XXX there's no such thing as "idle" in most event loops; this fakes
         # it the same way as Twisted, by scheduling the callback to be called
         # repeatedly
```

Comment:
update comment?

Reply:
IMHO it's still correct comment: we schedule callback via `loop.call_later`